src/deploy/BaseDeployer.js

```diff
@@ -27,6 +27,23 @@ class BaseDeployer {
   get relZip() {
     return path.relative(process.cwd(), this._builder.zipFile);
   }
+
+  get host() {
```

Comment:
I don't think this is a good idea. 
the problem is, that the host is only known, once the action is deployed. but the host is dependent on the api gateway selected. which remains (should remain) stable. so better deduce the host from the gatway key or read it from AWS.

Reply:
I will update the AWSDeployer accordingly

Reply:
Fixed in 22cedb7